packages/jsii/test/deprecated-remover.test.ts

```diff
@@ -0,0 +1,161 @@
+import { compileJsiiForTest } from '../lib';
+
+const DEPRECATED = '/** @deprecated stripped */';
+
+test('produces correct output', async () => {
```

Comment:
Would've been nicer to organize these fixtures and expectations as actual folders with files. 
Would be much easier to add new cases as we fix or add features to the remove-deprecated.

WDYT?

Reply:
Expectations as actual folders with files means we have to write our own way to update snapshots (`jest -u` will only update `.ts.snap` files, and the structure there is not very flexible). Also - how is more files to know about and modify, better than the whole GIVEN/WHEN/THEN is in a single place?